code/modules/clothing/shoes/jobs.dm

```diff
@@ -28,12 +28,18 @@
 	icon_state = "kneeboots"
 	item_state = "kneeboots"
 
+/obj/item/clothing/shoes/jackboots/knee/handle_movement(var/turf/walking, var/running)
+	trip_up()
```

Comment:
These aren't heels, they're boots that reach higher. There is no conceivable reason you would trip while wearing them.

Reply:
i'll give you this one